x/docs/md/intro/getting-started.md

```diff
@@ -3,7 +3,7 @@
 1. Download and configure the [command-line client](/cli).
 1. Fetch the first exercise (`exercism fetch TRACK_ID`).
 1. Write a solution to make the test suite pass.
-1. Submit your solution (`exercism submit path/to/file`).
+1. Submit your solution (`exercism submit path/to/file` or `exercism s path/to/file{.h,.cpp}`). 
```

Comment:
For test files you can you add filename{.h, .cpp, Test.cpp} or what ever the name of the file is. The client actually asks you if you want to send the test file which is really cool.